go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -273,12 +262,12 @@ func ensureClusterRaftMembership(db string, kclient kube.Interface) error {
 		}
 		if !memberFound && (len(members)-kickedMembersCount) > 3 {
 			// unknown member and we have enough members its safe to kick the unknown address
-			klog.Infof("Unknown Raft member found in %s: %s, %s... kicking", db, member[1], member[3])
-			_, stderr, err = appCtl(5, "cluster/kick", dbName, member[1])
+			klog.Infof("Unknown Raft member found in %s: %s, %s... kicking", db.dbName, member[1], member[3])
+			_, stderr, err = db.appCtl(5, "cluster/kick", db.dbName, member[1])
 			if err != nil {
 				// warn only: we might fail to kick since other nodes will also be trying to kick the member
 				klog.Warningf("Error while kicking old Raft member: %s, for address: %s in db: %s,"+
-					"stderr: %v, err: %v", member[1], member[3], db, stderr, err)
+					"stderr: %v, err: %v", member[1], member[2], db.dbAlias, stderr, err)
```

Comment:
@andreaskaris is it intentional here to change member[3] to member[2] ?

Reply:
Nope. Typo. me~~ ; I fixed it and made the tests run into this path of the code now, too:
 https://github.com/ovn-org/ovn-kubernetes/pull/2582/commits/8181b0ae2f5fae8b15f8524ed91e8fdf8f1c3a77#diff-6eecb13689493869d78a0b346a91937e693d3d4c97976b1d2aa431c964590656R308
That doesn't change anything to the outcome, but the message will be printed when running the tests.

Reply:
[root@ovnkubernetes go-controller]# go test -count 1 -mod=vendor -test.v -race  github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovndbmanager
I0105 11:42:33.961490  104349 ovs.go:93] Maximum command line arguments set to: 191102
=== RUN   TestEnsureLocalRaftServerID
=== RUN   TestEnsureLocalRaftServerID/0:Test_error:_unable_to_get_db_server_ID
=== RUN   TestEnsureLocalRaftServerID/1:Test_error:_Invalid_sid
=== RUN   TestEnsureLocalRaftServerID/2:Test_error:_Unable_to_get_cluster_status
=== RUN   TestEnsureLocalRaftServerID/3:Test_error:_unable_to_parse_address_for_db
=== RUN   TestEnsureLocalRaftServerID/4:Test_error:_while_kicking_old_Raft_member
I0105 11:42:33.970319  104349 ovndbmanager.go:178] Previous stale member found in OVN_Northbound: 3936... kicking
=== RUN   TestEnsureLocalRaftServerID/5:Stale_member,_kick
I0105 11:42:33.971577  104349 ovndbmanager.go:178] Previous stale member found in OVN_Northbound: 3936... kicking
=== RUN   TestEnsureLocalRaftServerID/6:Consistent_database,_no_action_needed
--- PASS: TestEnsureLocalRaftServerID (0.00s)
    --- PASS: TestEnsureLocalRaftServerID/0:Test_error:_unable_to_get_db_server_ID (0.00s)
    --- PASS: TestEnsureLocalRaftServerID/1:Test_error:_Invalid_sid (0.00s)
    --- PASS: TestEnsureLocalRaftServerID/2:Test_error:_Unable_to_get_cluster_status (0.00s)
    --- PASS: TestEnsureLocalRaftServerID/3:Test_error:_unable_to_parse_address_for_db (0.00s)
    --- PASS: TestEnsureLocalRaftServerID/4:Test_error:_while_kicking_old_Raft_member (0.00s)
    --- PASS: TestEnsureLocalRaftServerID/5:Stale_member,_kick (0.00s)
    --- PASS: TestEnsureLocalRaftServerID/6:Consistent_database,_no_action_needed (0.00s)
=== RUN   TestEnsureClusterRaftMembership
=== RUN   TestEnsureClusterRaftMembership/0:Test_error:_Invalid_database_name
=== RUN   TestEnsureClusterRaftMembership/1:Test_error:_Unable_to_get_cluster_status
=== RUN   TestEnsureClusterRaftMembership/2:Consistent_database,_no_action_needed
**=== RUN   TestEnsureClusterRaftMembership/3:Unknown_Raft_member,_kick**
**I0105 11:42:33.977329  104349 ovndbmanager.go:265] Unknown Raft member found in OVN_Northbound: c10c, 10.1.1.219:9643... kicking**
**I0105 11:42:33.977382  104349 ovndbmanager.go:265] Unknown Raft member found in OVN_Northbound: fc43, 10.1.1.220:9643... kicking**
**W0105 11:42:33.977408  104349 ovndbmanager.go:269] Error while kicking old Raft member: fc43, for address: 10.1.1.220:9643 in db: ovnnb,stderr: unknown server**
**ovn-appctl: /var/run/ovn/ovnnb_db.ctl: server returned an error, err: Error unkown server**
**--- PASS: TestEnsureClusterRaftMembership (0.00s)**
    --- PASS: TestEnsureClusterRaftMembership/0:Test_error:_Invalid_database_name (0.00s)
    --- PASS: TestEnsureClusterRaftMembership/1:Test_error:_Unable_to_get_cluster_status (0.00s)
    --- PASS: TestEnsureClusterRaftMembership/2:Consistent_database,_no_action_needed (0.00s)
**    --- PASS: TestEnsureClusterRaftMembership/3:Unknown_Raft_member,_kick (0.00s)**
=== RUN   TestEnsureElectionTimeout
=== RUN   TestEnsureElectionTimeout/0:Test_error:_Unable_to_get_cluster_status
=== RUN   TestEnsureElectionTimeout/1:Test_error:_Failed_to_get_current_election_timer
=== RUN   TestEnsureElectionTimeout/2:Follower,_not_trying_to_change
=== RUN   TestEnsureElectionTimeout/3:leader,_timer_doesn't_change
=== RUN   TestEnsureElectionTimeout/4:Test_error:_failed_to_change_election_timer_when_leader_timer_must_change
=== RUN   TestEnsureElectionTimeout/5:Test_error:_failed_to_change_election_timer_when_leader_timer_must_change_but_desired_is_more_than_double
=== RUN   TestEnsureElectionTimeout/6:leader,_timer_must_change
=== RUN   TestEnsureElectionTimeout/7:leader,_timer_must_change_but_desired_is_more_than_double
--- PASS: TestEnsureElectionTimeout (0.00s)
    --- PASS: TestEnsureElectionTimeout/0:Test_error:_Unable_to_get_cluster_status (0.00s)
    --- PASS: TestEnsureElectionTimeout/1:Test_error:_Failed_to_get_current_election_timer (0.00s)
    --- PASS: TestEnsureElectionTimeout/2:Follower,_not_trying_to_change (0.00s)
    --- PASS: TestEnsureElectionTimeout/3:leader,_timer_doesn't_change (0.00s)
    --- PASS: TestEnsureElectionTimeout/4:Test_error:_failed_to_change_election_timer_when_leader_timer_must_change (0.00s)
    --- PASS: TestEnsureElectionTimeout/5:Test_error:_failed_to_change_election_timer_when_leader_timer_must_change_but_desired_is_more_than_double (0.00s)
    --- PASS: TestEnsureElectionTimeout/6:leader,_timer_must_change (0.00s)
    --- PASS: TestEnsureElectionTimeout/7:leader,_timer_must_change_but_desired_is_more_than_double (0.00s)
=== RUN   TestResetRaftDB
=== RUN   TestResetRaftDB/0:Failed_to_back_up_the_db_to_backupFile
=== RUN   TestResetRaftDB/1:Failed_to_restart_the_database
I0105 11:42:33.981277  104349 ovndbmanager.go:331] Backed up the db to backupFile: ovnnb2022-01-05_114233db_bak
=== RUN   TestResetRaftDB/2:Successful_database_backup
I0105 11:42:33.981729  104349 ovndbmanager.go:331] Backed up the db to backupFile: ovnnb2022-01-05_114233db_bak
I0105 11:42:33.981759  104349 ovndbmanager.go:337] Stopped OVN_Northbound db after backing up the db: ovnnb2022-01-05_114233db_bak
--- PASS: TestResetRaftDB (0.00s)
    --- PASS: TestResetRaftDB/0:Failed_to_back_up_the_db_to_backupFile (0.00s)
    --- PASS: TestResetRaftDB/1:Failed_to_restart_the_database (0.00s)
    --- PASS: TestResetRaftDB/2:Successful_database_backup (0.00s)
PASS
ok  	github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovndbmanager	0.401s